src/page.rs

```diff
@@ -29,6 +37,13 @@ pub struct Page<T> {
     pub last: Option<Url>,
 }
 
+#[cfg(feature = "stream")]
```

Comment:
This doesn’t need a cfg because it doesn’t use any stream specific stuff

Reply:
But if we don't put the cfg on it we'll get a dead code warning, which I wanted to avoid.

Reply:
Ah yeah, then it's fine.